voorhoede/voorhoede-website
# feat/video-cookie-notice

## What changes were made

- adds generic cookie notice component 
- add cookie notice to responsive-video component

the cookie notice is shown if you have not accepted the cookies for the video, if you have accepted them, they are saved in local storage and retrieved the next time (so you do not see the cookie notice again). Users can now only view the videos if they agree with the cookie-notice

## Ticket

https://trello.com/c/V1pCpO5N/634-cookies-yt-video-on-website

## How to test or check results



check deploy preview (the `/:locale/cases/quantum-inspire/` page has videos on it)

## Checks
